packages/itwinui-react/src/core/Buttons/Button.test.tsx

```diff
@@ -220,3 +221,25 @@ it('should respect `stretched` prop', () => {
   const { container } = render(<Button stretched>Do not click</Button>);
   expect(container.querySelector('button')).toHaveStyle('--_iui-width: 100%');
 });
+
+it.each(['default', 'small', 'large'] as const)(
```

Comment:
nit/optional: Maybe this test can also confirm that the label is still present in the DOM.

Reply:
Nope, it's not important here. There will be proper tests for the semantics later.